### PR TITLE
Require only the bits of Rails we need now

### PR DESCRIPTION
**Pull request by mjgiarlo**
# Why was this change made? 🤔

Simplicity.

# How was this change tested? 🤨

CI + local testing

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+require "action_cable/engine"
 require "action_controller/railtie"
+# require "action_mailbox/engine"
 require "action_mailer/railtie"
-require "action_mailbox/engine"
-require "action_text/engine"
+# require "action_text/engine"
 require "action_view/railtie"
-require "action_cable/engine"
+# require "active_job/railtie"
```
Comment (justinlittman) [inline config/application.rb:10]: Are we going to forget that we turned off ActiveJob when we need it later?

**Review by justinlittman — APPROVED** [review]
Approve pending consideration of comment.

File: config/application.rb (hunk 1 above)
Comment (mjgiarlo) [reply config/application.rb:10]: Possibly! Since we're likelier to use AJ over these other components, shall I flip that one on now?

Comment (mjgiarlo) [reply config/application.rb]: @justinlittman Done.

**Pull request merged**